lineman/app/modules/inbox/inbox.coffee

```diff
@@ -3,6 +3,19 @@ angular.module('loomioApp').directive 'inbox', ->
   restrict: 'E'
   templateUrl: 'generated/modules/inbox/inbox.html'
   replace: true
-  controller: ($scope, Records, UserAuthService) ->
-    $scope.currentUser = UserAuthService.currentUser
-    Records.discussions.fetchInbox()
+  controller: ($scope, InboxService) ->
+    # build a loki view for each of these modes.
+    # maybe within the service
+    InboxService.fetchRecords().then (data) ->
+      $scope.unreadThreads = InboxService.unreadThreads()
+      $scope.currentThreads = InboxService.currentThreads()
+
+    $scope.unreadThreads = InboxService.unreadThreads()
```

Comment:
I'm confused by this; why call these twice? And why pass data through the fetchRecords promise?